test/src/com/xilinx/rapidwright/TestCheckOpenFilesInstalled.java

```diff
@@ -27,12 +27,18 @@
 
 import com.xilinx.rapidwright.support.CheckOpenFilesExtension;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIf;
 import org.junit.jupiter.api.extension.ExtendWith;
 
 public class TestCheckOpenFilesInstalled {
     @Test
+    @EnabledIf("runningInGradle")
     @ExtendWith(CheckOpenFilesExtension.CheckOpenFilesWorkingExtension.class)
     public void test() throws IOException {
         //Actual test is in extension
     }
+
+    private boolean runningInGradle() {
+        return System.getProperty("junit.jupiter.extensions.autodetection.enabled") != null;
```

Comment:
Pedantic:
```suggestion
        return System.getProperty("junit.jupiter.extensions.autodetection.enabled").equals("true");
```

Reply:
I don't think this works when running outside of gradle; since `null.equals("true")` gives the error: "Cannot invoke equals(String) on the primitive type null"

Reply:
Ooh, yes! Try the other way around:
```
return "true".equals(System.getProperty("junit.jupiter.extensions.autodetection.enabled"));
```